### ISSUE TITLE
Click here if the image fails loading

### ISSUE THREAD

**Issue by josh56424** [hanFengSan/eHunter#25]
button for reload

**Comment by hanFengSan** [comment]
click where? I put two or three text button in center for reloading.

**Comment by josh56424** [comment]
how to load  images 1 by 1 
loading is not smooth even selecting 1 per page 
example start reading manga page 1 t o page 8 fine but after that whole thing is mess up i have click on reload image  after multiple page stop loading

what i am trying to say is that script is not loading script one by one and the pages are not loading until i scroll down. its much better if pages load automatically ex.from 1 to 100 because probably that the reason is that page loading is not smooth and   

i want load pages properly one by one 
for example opened the gallery and total are  100 page if script start loading from 1st page then 2nd page then 3rd page if failed the fourth image auto reload the image using 3rd refreshing option ( from Other source) that works the best and fastest do not load next image until 4 page is complete .

in short
prob 
1-script is not loading script one by one and the pages are not loading until i scroll down. (this problem really pain please solve it )
2- load page automatically ex.from 1 to 100 because probably that the reason is that page loading is not smooth.

3-auto refresh the page 
thanx for your hard work. your script is only script that work with bigger thumbnail you are awesome man.
 
great work

**Comment by hanFengSan** [comment]
Do you see the 'Loading quantity'? The images will be auto load from “your current location - Loading quantity” to  “your current location + Loading quantity”.  If you in 'Book' mode, it will be convert to screens.
It isn't  designed to load 1 by 1,  just want to preload some images for a better experience.
I see your problem, and I think that I just need to improve the auto reloading if failed. What do you think about it?

**Comment by josh56424** [comment]
this script auto retry to download images if failed  for reference
https://sleazyfork.org/en/scripts/10379-e-hentai-downloader

**Comment by hanFengSan** [comment]
Thx, it's very easy to do. I don't have time to do it now, so it will be done after some weeks.

**Comment by josh56424** [comment]
script and extension that work for exhentai i tried
this extension was the best but do not refresh images 
and force images to load one by one without scroll in korea mode  
manga viewer
version 15 work with exhentai 
https://www.crx4chrome.com/extensions/bebalkdfejapnfbngpmhchkboajaofen/

3-this one is  similar to yours

**Comment by josh56424** [comment]
thankx

**Comment by hanFengSan** [comment]
It's good😆
